.github/workflows/valkyrie.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         run: |
           bash scripts/gh-action-unit-test.sh ${GITHUB_WORKFLOW}
       - name: Run integration tests
+        env:
```

Comment:
do we need divo_down disabled for github action executions? I expected to only utilize it for local work.

if we do need it in the actions, why specifically valkyrie?